recipes/docling/recipe.yaml

```diff
@@ -0,0 +1,80 @@
+context:
+  name: docling
+  version: 2.6.0
+  python_min: "3.10"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/docling-${{ version }}.tar.gz
+  sha256: 282755472a1a1d4a4088d794c6a89f399a5b7fbcea8b5f45012c441670b538d5
+  patches:
+    # lower the scipy lower bounds to match with c-f and avoid conflicts
+    - fix-deps.patch
+
+build:
+  number: 0
+  noarch: python
```

Comment:
> the win failure is due to missing downstream deps.

Not sure if we have diverging nomenclature here. What is "downstream" supposed to mean? in any case, given that this depends on `deepsearch-glm` (which is `skip: win`) and is noarch, the windows failures should be fine.

Reply:
Yes sorry I got it wrong :-)

Indeed the failure is fine.